tests/test_base.py

```diff
@@ -37,7 +37,7 @@ def test_passwd_present(auto_container):
 
 @pytest.mark.skipif(
     OS_VERSION not in ("tumbleweed",),
-    reason="requires glibc-locale-base installed",
+    reason="requires gconv modules installed",
```

Comment:
```suggestion
    reason="requires gconv modules to be installed",
```

Reply:
I wanted shorter, not longer. I've removed installed now.